internal/drivers/go-duckdb.go

```diff
@@ -0,0 +1,9 @@
+package drivers
+
+// implements the pgx driver https://github.com/marcboeker/go-duckdb
+// func init() {
+// 	genieql.RegisterDriver(PGX, NewDriver("github.com/marcboeker/go-duckdb", ddbexports(), ddb...))
+// }
+
+// DDB - driver for github.com/marcboeker/go-duckdb
+const DDB = "github.com/marcboeker/go-duckdb"
```

Comment:
should be understandable to someone glancing at the const with no context.
```suggestion
const DuckDB = "github.com/marcboeker/go-duckdb"
```